smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/integration/HTTPProtocolCustomizable.kt

```diff
@@ -9,10 +9,11 @@ import software.amazon.smithy.codegen.core.Symbol
 import software.amazon.smithy.model.shapes.OperationShape
 import software.amazon.smithy.model.shapes.ServiceShape
 import software.amazon.smithy.model.shapes.Shape
+import software.amazon.smithy.model.traits.TimestampFormatTrait
 import software.amazon.smithy.protocoltests.traits.HttpRequestTestCase
 import software.amazon.smithy.swift.codegen.SwiftWriter
 
-interface HttpProtocolCustomizable {
+interface HTTPProtocolCustomizable {
```

Comment:
Renamed, and added some customization points that used to be passed around piecemeal as params between codegen components.